doc/api/fs.md

```diff
@@ -3918,6 +3918,33 @@ On Linux, positional writes do not work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+#### filehandle.write(string[, position[, encoding]])
+<!-- YAML
+added: v10.0.0
+-->
+
+* `string` {string}
+* `position` {integer}
+* `encoding` {string} **Default:** `'utf8'`
+* Returns: {Promise}
+
+Write `string` to the file. If `string` is not a string, then
+the value will be coerced to one.
+
+`position` refers to the offset from the beginning of the file where this data
+should be written. If `typeof position !== 'number'` the data will be written at
+the current position. See pwrite(2).
+
+`encoding` is the expected string encoding.
+
+It is unsafe to use `filehandle.write()` multiple times on the same file
+without waiting for the `Promise` to be resolved (or rejected). For this
```

Comment:
Nit: a rejected promise is still resolved (and so is a promise following another promise) - `fulfilled` instead of resolved or "settled" instead of either would work here.

Reply:
Hey there, I've addressed some other feedback but I haven't addressed this one yet. I did that for two reasons.

1. This is the same wording used in other parts of the documentation. See the docs for the [filehandle.write(buffer)](https://github.com/nodejs/node/blob/1532232dd95f58378056864b6d1294d45b80c82d/doc/api/fs.md#filehandlewritebuffer-offset-length-position)
2. I would argue that explicitly using the words `resolved` and `rejected` make sense in this context. If you're familiar with Promises, this makes sense. I would argue the same cannot exactly be said about the word `fulfilled` because it's not as commonly used when referring to Promises. (at least as far as I can tell). In my opinion that potentially introduces some ambiguity.

All that being said: I am happy to make the suggested changes here, as well as in the other parts of the same doc to ensure consistency.